src/pubsub.js

```diff
@@ -69,6 +69,22 @@
         subscriber( message, data );
     }
 
+    function findWildCardMessage(topic)
+    {
+        var foundMessages = Object.keys(messages).find(function(messageItem){
+            return messageItem.indexOf('*') > -1 && matchMessage(topic,messageItem);
+        });
+        return foundMessages || []; 
+    }
+
+    function matchMessage(str, rule) {
+        var escapeRegex = (str) => str.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, "\\$1");
+        rule = rule.split("*").map(escapeRegex).join(".*");
```

Comment:
`.map` on Array was introduced in ES5, so you'll need to find another solution for this too.